queries.sql

```diff
@@ -82,3 +82,178 @@ SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+ 10 | Blossom    | 1998-10-13    |               3 | t       |        17 | unspecified
+ 11 | Ditto      | 2022-05-14    |               4 | t       |        22 | unspecified
+(11 rows)
+*/
+
+ROLLBACK;
+SELECT * FROM animals;
+/* 
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg |   species
+----+------------+---------------+-----------------+---------+-----------+--------------
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 |
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 |
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 |
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 |
+  5 | Charmander | 2020-02-08    |               0 | f       |    -11.00 | Fire Lizard
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | Plant
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 | Water Turtle
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | Angel
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | Boar
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 | Flower
+ 11 | Ditto      | 2022-05-14    |               4 | t       |     22.00 | Transforming
+(11 rows)
+*/
+
+START TRANSACTION;
+
+-- Update the species column to "digimon" for animals whose names end in "mon"
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Update the species column to "pokemon" for animals whose species is not set
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+/*
+ id |    name    | date_of_birth | escape_attempts | neutred | weight_kg |   species
+----+------------+---------------+-----------------+---------+-----------+--------------
+  5 | Charmander | 2020-02-08    |               0 | f       |    -11.00 | Fire Lizard
+  7 | Squirtle   | 1993-04-02    |               3 | f       |    -12.13 | Water Turtle
+ 10 | Blossom    | 1998-10-13    |               3 | t       |     17.00 | Flower
+ 11 | Ditto      | 2022-05-14    |               4 | t       |     22.00 | Transforming
+  3 | Pikachu    | 2021-01-07    |               1 | f       |     15.04 | pokemon
+  1 | Agumon     | 2020-02-03    |               0 | t       |     10.23 | digimon
+  2 | Gabumon    | 2018-11-15    |               2 | t       |      8.00 | digimon
+  4 | Devimon    | 2017-05-12    |               5 | t       |     11.00 | digimon
+  6 | Plantmon   | 2021-11-15    |               2 | t       |     -5.70 | digimon
+  8 | Angemon    | 2005-06-12    |               1 | t       |    -45.00 | digimon
+  9 | Boarmon    | 2005-06-07    |               7 | t       |     20.40 | digimon
+(11 rows)
+*/
+
+-- Start the transaction
+START TRANSACTION;
+
+-- Delete all records from the animals table
+DELETE FROM animals;
+
+-- Roll back the transaction to undo the delete
+ROLLBACK;
+
```

Comment:
As per the project requirements, after the rollback here, please add another statement to verify if the animals' table still exists🙏

Reply:
- [ ] Kindly address this part of the comment
![Screenshot from 2023-07-12 09-49-10](https://github.com/rachaouldbabaali/vet-clinic-db/assets/46388113/d7bffab0-e120-4f45-9722-e8ca9696236e)